app/page.tsx

```diff
@@ -85,7 +90,11 @@ export default async function Index() {
             </>
           ),
           icon: <Clock />,
-          value: prettyMs(Number(recentSummary.avg_proving_time) || 0),
+          value: (
+            <HidePunctuation>
+              {prettyMs(Number(recentSummary.avg_proving_time) || 0)}
+            </HidePunctuation>
```

Comment:
Not sure if we should so this in this case where we have decimals
![image](https://github.com/user-attachments/assets/ca41c186-c359-423d-bf72-cbbcb5a34c2e)

Reply:
cc @nloureiro

Reply:
Okay sure, this may not be worth it... was a small attempt to trim the spacing here, but this could complicate it a touch unless I just ignore decimal points as well.

Reply:
Patched